src/components/InstallerView.jsx

```diff
@@ -0,0 +1,31 @@
+import viewStyles from '../styles/view'
+import React, { Component } from 'react'
+import { translate } from 'cozy-ui/react/helpers/i18n'
+import classNames from 'classnames'
+
+class InstallerView extends Component {
+  doInstall () {
+    // get the values from the fields
+    const slug = document.querySelector('#slug').value
```

Comment:
We try to avoid querying the DOM in component. You may instead use a stateful Component :
```javascript
class InstaerView extends Component {
  constructor (props) {
    super(props)
    this.state = {
      slug: null,
      url: null
   }

   render (props) {
     return(
      ...
      <input [...] name="slug" onChange={(event) => this.setState({slug: event.target.value})} />
      ...
  }
```
Then, in `doInstall`, we just have to rely on component's state.

@goldoraf May you confirm please ?